core/trino-main/src/main/java/io/trino/sql/planner/InputExtractor.java

```diff
@@ -83,6 +83,10 @@ public Set<Input> getInputs()
         @Override
         public Void visitTableScan(TableScanNode node, PlanFragmentId fragmentId)
         {
+//            if (node.isUpdateTarget()) {
```

Comment:
left-over?

Reply:
Yes, it was intended to catch the error first.